libs/media-player/src/lib/MediaPlayer.tsx

```diff
@@ -1,16 +1,13 @@
-import PauseCircleFilledIcon from '@mui/icons-material/PauseCircleFilled';
-import PlayCircleFilledIcon from '@mui/icons-material/PlayCircleFilled';
+import PauseIcon from '@mui/icons-material/Pause';
+import PlayArrowIcon from '@mui/icons-material/PlayArrow';
+import { Box, IconButton } from '@mui/material';
 import { useEffect, useState } from 'react';
-import styles from './MediaPlayer.module.scss';
 
 export interface MediaPlayerProps {
     audioUrl: string;
-    listenMessage?: string;
```

Comment:
You are causing regression on Justin and Blake's radio page for the language hub. They should have a test that breaks because of this.